src/mono/Directory.Build.targets

```diff
@@ -14,4 +14,29 @@
     <InformationalVersion>$(ProductVersion)</InformationalVersion>
     <InformationalVersion Condition="'$(VersionSuffix)' != ''">$(InformationalVersion)-$(VersionSuffix)</InformationalVersion>
   </PropertyGroup>
+
+  <!--
+    SDK currently does not know about net6.0 TargetFramework.
```

Comment:
Assuming that these item groups are all identical across libraries, mono and installer, could we move them into a separate props file and import it conditionally in the places where we need it? The condition could we written in a way that the file is automatically imported when we use a tfm that the SDK doesn't yet support.